<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showCourseInfo</class>
 <widget class="QDialog" name="showCourseInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="CourseInfoEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>333</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="homeworkEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>341</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="materialEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>333</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>课程信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作业</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>60</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>课程资料</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeworkButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>上传作业</string>
   </property>
  </widget>
  <widget class="QPushButton" name="materialButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>上传资料</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
